### DISCUSSION TITLE
kube-teleport-agent does not start - only shows ready

### DISCUSSION THREAD

**Discussion by MysterMyner** [gravitational/teleport#31917] [Q&A]
In a nutshell:

- I've deployed a VPC on AWS with a four ec2 vms - three amazon linux and one ubuntu 22.04
- The same VPC security group applied to all VM's allows for complete TCP inbound visibility to the entire vpc cidr range. Additionally inbound access to port 80 and 443 is open to the world (0.0.0.0/0). Only the TP proxy host has a public IP... the others all have private IPs. The single SG also allows for unfettered outbound access to all VMs
- TP proxy deploys just fine on the first amz linux ec2 - using it with a letsencrypt cert on a google domain i rent. Simple enough to run this and add my first admin user
- I can deploy docker on one of the amz linux and fire up a number of webapps like portainer, focalboard
- I can add them to teleport and access works great through the TP proxy webui
- I can add the second linux machine as a target server and access it via the in-browser SSH screen
- In both cases I'm using the documented "getting started" workflows - generate a token on the proxy/auth host and use that to start teleport agents on the respective boxes
- I've run the third ubuntu machine with microk8s (with rbac enabled) and then tried with minikube. In both cases I can create the long lived join token (TTL set to 15 days) and then deploy the teleport kube agent with helm. Over multiple trials, I've tried both the docs CLI approach as well as the updated "add from the TP proxy webui method"
    - The kube deployment runs just fine - I see 
    - `ubuntu@ip-10-1-0-190:~$ kubectl get pods -n tp`
    - `NAME               READY   STATUS    RESTARTS   AGE`
    - `teleport-agent-0   0/1     Running   0          30m`
- The pod never goes to `1/1` - stays at 0 no matter how long. The kube cluster never shows up in the TP Proxy WebUI either... times out eventually
- Given that I just tried this with two completely separate distros of kube, makes me think that there's a step or network config that I'm missing
_Would appreciate any help to get this running. This isn't mission critical - I'm just trying to play with this really seemingly cool tech, but unable to make this work_

**Comment by AntonAM** [comment]
You should be able to see more details if you run
```
kubectl -n tp logs teleport-agent-0
```
